tests/e2e/pageobjects/ide/Ide.ts

```diff
@@ -166,7 +166,7 @@ export class Ide {
         }
     }
 
-    async waitLeftToolbarButton(buttonTitle: LeftToolbarButton, timeout: number = TimeoutConstants.TS_SELENIUM_TOOLBAR_TIMEOUT) {
```

Comment:
IMHO it would be better not to change TS_SELENIUM_TOOLBAR_TIMEOUT constant name in here, because it has quite a relevant semantic related to IDE.
Alternatively we can increase TS_SELENIUM_TOOLBAR_TIMEOUT timeout number in TimeoutConstants file.